### ISSUE TITLE
Strain amp dev branch necessary?

### ISSUE THREAD

**Issue by tkevinbest** [neurobionics/opensourceleg#142]
Is the strain amp dev branch stale? Let's delete it if no one is using it anymore. I think everything in it was merged into main.

**Comment by imsenthur** [comment]
Yes--It has some of Mitry's TMotor code in it but at this point it is so far behind the master and dev branches so I'll go ahead and delete it.
